docs/usage/syncing-instances.md

```diff
@@ -2,11 +2,60 @@
 sidebar_position: 4
 ---
 # Syncing Instances
-So, how to sync every instance of HuronOS at the same time?, you might ask. The answer is: using a directives file. Where to retrieve this file from is defined when a HuronOS usb is created, and that file is retrieved every minute for every HuronOS instance (although that might not happen at the same time) so, if the directives file url is correctly configured and set on the usbs, every instance should be in sync with the directives file wherever that file is stored.
 
+**Remote syncing** is one of the flagship features of huronOS, allowing you to synchronize preferences, configurations, and behavior directives with **several huronOS instances** (running installations of huronOS) using something we call *the directives file*. This file is a plain text file with a specific syntax known as *huronOS Directives File Syntax*, resulting in a *.hdf* file.
```

Comment:
After mentioning the directives file, to avoid redundancies: [...].This is a plain text file [...]